cfgov/jinja2/v1/_layouts/spanish-ask-base.html

```diff
@@ -1,200 +1,26 @@
-                <li><a href="http://flickr.com/cfpbphotos" hreflang="en-US"><img src="{{ static("nemo/_/img/icon-flickr-retina.png") }}" alt="Flickr" class="icon-25"></a></li>
-                <li><a href="http://youtube.com/CFPB" hreflang="en-US"><img src="{{ static("nemo/_/img/icon-youtube-retina.png") }}" alt="Youtube" class="icon-25"></a></li>
-            </ul>
-            <p class="s-show-on-small"><small>Un sitio web oficial del gobierno federal de <span class="usa">los Estados Unidos <img src="{{ static('nemo/_/img/us_flag_small.png') }}" alt=""></span></small></p>
-            <p><a class="a-btn btn-block-on-small english-link" href="/" hreflang="en-US">English</a></p>
-        </nav>
-        <div class="js-showtoggle policy">
-            <button class="a-btn btn-block-on-small js-showtoggle-trigger">Política de información</button>
-            <div class="js-showtoggle-content">
-                <div class="row">
-                    <div class="span4">
-                        <p>En este momento, la política de información sólo está disponible en inglés.</p>
-                    </div>
-                    <div class="span4">
-                        <ul class="unstyled">
-                            <li><a href="/privacy-policy/" hreflang="en-US">Privacy policy and legal notices</a></li>
-                            <li><a href="/accessibility/" hreflang="en-US">Accessibility</a></li>
-                            <li><a href="/plain-writing/" hreflang="en-US">Plain writing</a></li>
-                            <li><a href="/no-fear-act/" hreflang="en-US">No FEAR Act</a></li>
-                        </ul>
-                    </div>
-                    <div class="span4">
-                        <ul class="unstyled">
-                            <li><a href="/foia/" hreflang="en-US">FOIA</a></li>
-                            <li><a href="http://gobiernousa.gov/" hreflang="es">GobiernoUSA.gov</a></li>
-                            <li><a href="http://www.federalreserve.gov/oig/default.htm" hreflang="en-US">Office of Inspector General</a></li>
-                            <li><a href="/ombudsman/" hreflang="en-US">Ombudsman</a></li>
-                        </ul>
-                    </div>
-                </div>
-            </div><!-- .js-showtoggle-content -->
-        </div><!-- .policy -->
-
-    </div><!-- .wrapper-container -->
-
-</footer><!-- .wrapper-footer -->
-
-<script src="{{ static('nemo/_/js/scripts.min.js') }}"></script>
-
-{% block page_js %}
-<!--[if gt IE 8]><!-->
-<script src="{{ static('js/spanish.js') }}"></script>
-<!--<![endif]-->
 {% endblock %}
 
+{% block nav_item_respuestas %}
+    s-active
+{% endblock %}
 
-
-</body>
-</html>
+{% block ask_search %}
+    <div class="ac-search full-bleed full-bleed-padded {% block babel_hide_search %}s-hide-on-small{% endblock %}">
```

Comment:
`babel_hide_search` could use a code comment above it too since it's hidden inline.